indexing-service/src/test/java/org/apache/druid/indexing/common/task/batch/parallel/TaskMonitorTest.java

```diff
@@ -195,6 +195,7 @@ public TaskStatusResponse getTaskStatus(String taskId)
       return new TaskStatusResponse(
           taskId,
           new TaskStatusPlus(
+              taskId,
```

Comment:
I guess it might be fine to use the `taskId` as the `groupId` since the [default value of `groupId` is the `taskId`](https://github.com/apache/incubator-druid/blob/master/indexing-service/src/main/java/org/apache/druid/indexing/common/task/AbstractTask.java#L75). But, since `TaskMonitor` is used to monitor sub tasks of the parallel indexing task, it could be more realistic to use the groupId of the taskId of the supervisor task.

Reply:
In this particular test, the groupId is always `"groupId"` as seen [here](https://github.com/apache/incubator-druid/blob/8a920ec9b9dec5fb5ac798807c7c5537a804b40a/indexing-service/src/test/java/org/apache/druid/indexing/common/task/batch/parallel/TaskMonitorTest.java#L80). You can use it if you want.